index.html

```diff
@@ -297,8 +296,59 @@ <h2>Contact me</h2>
         </section>
     </section>
 
-    <script src="./JS/"></script>
-    <script src="./JS/"></script>
+    <!-- Modal -->
+    <section class="modalContainer" id="modalContainers">
+      <div class="modalHeader">
+          <div class="modalNav">
+            <h3 class="card-title" id="modalTitle"></h3>
+            <div class="modalNavLi" id="close-btn">&times;
+
+            </div>
+          </div>
+          <ul class="canopy" id="canopy">
+            <!-- <li class="canopyLi1">CANOPY</li>
```

Comment:
Kindly don't leave any uncommented code in your codebase. In this way, you can keep your codebase clean, organized and professional 👍 If you need these codes later you can travel through your history with git. 📝 Please don't forget to remove the uncommented code in index.html as well 👍